parm/data_locations.yml

```diff
@@ -450,16 +320,22 @@ CCPA_obs:
 
 MRMS_obs:
   hpss:
-    <<: *MRMS_hpss
-    file_names:
-      obs:
-        - "MergedReflectivityQCComposite_00.50_{yyyymmdd}-*.grib2.gz"
-        - "EchoTop_18_00.50_{yyyymmdd}-*.grib2.gz"
+    protocol: htar
+    archive_format: tar
+    archive_path:
```

Comment:
This is missing the `archive_file_names:` entry from the old `NSSL_mrms` stanza above.

Reply:
Ooh. Good catch. Thanks.